lib/spack/spack/main.py

```diff
@@ -363,7 +363,15 @@ def make_argument_parser(**kwargs):
         metavar='DIR', help="add a custom configuration scope")
     parser.add_argument(
         '-d', '--debug', action='store_true',
-        help="write out debug logs during compile")
+        help="write out basic debug messages")
+    parser.add_argument(
+        '-dd', action='store_true',
+        help="write out basic and standard debug messages")
+    parser.add_argument(
+        '-ddd', action='store_true',
+        help="write out basic, standard and detailed debug messages")
+    parser.add_argument(
+        '-dddd', action='store_true', help="write out all debug messages")
```

Comment:
```suggestion
    parser.add_argument('-d', '--debug', action='count', default=0, 
                                        help="increase debug level. '-dd' is standard debug output, '-dddd' is maximum debug output.")
    
```

Reply:
I think it's a bit confusing if `-dd` is "standard".  I don't think we need to say that -- just say more d's for more debug output.

Reply:
Removed.